pkg/crc/preflight/preflight.go

```diff
@@ -153,6 +153,11 @@ func doRegisterSettings(cfg config.Schema, checks []Check) {
 func StartPreflightChecks(config config.Storage) error {
 	experimentalFeatures := config.Get(cmdConfig.ExperimentalFeatures).AsBool()
 	mode := network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString())
+	if mode == network.VSockMode {
```

Comment:
Is there a reason to not use a regular preflight check for this ?

Reply:
@guillaumerose then we need to add a network check on the mac since there is no network checks on the mac side.